linter.py

```diff
@@ -10,12 +10,11 @@
 
 """This module exports the Flake8 plugin linter class."""
 
-import os
-from SublimeLinter.lint import persist, PythonLinter
+from SublimeLinter.lint import PythonLinter
 
 
 class Flake8(PythonLinter):
-
+    
```

Comment:
This is the only line causing the travis failure:

```
The command "flake8 . --max-line-length=120" exited with 1.
$ pep257 . --ignore=D202
./linter.py:16 in public class `Flake8`:
        D211: No blank lines allowed before class docstring (found 1)
```

Can you remove this so the build passes?  I'd love to see this merged :+1: